### ISSUE TITLE
TypeError: Passing coroutines is forbidden, use tasks explicitly.

### ISSUE THREAD

**Issue by kesmit13** [P403n1x87/austin-tui#28]
### Description

I'm running Austin-tui 1.2.2 with Python 3.11.3. Any time I try to use any keyboard navigation it crashes with the following:
```
Traceback (most recent call last):
  File "/home/me/.pyenv/versions/3.11.3/bin/austin-tui", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/me/.pyenv/versions/3.11.3/lib/python3.11/site-packages/austin_tui/__main__.py", line 175, in main
    tui.run()
  File "/home/me/.pyenv/versions/3.11.3/lib/python3.11/site-packages/austin_tui/__main__.py", line 140, in run
    self._view.close()
  File "/home/me/.pyenv/versions/3.11.3/lib/python3.11/site-packages/austin_tui/view/__init__.py", line 248, in close
    task.result()
  File "/home/me/.pyenv/versions/3.11.3/lib/python3.11/site-packages/austin_tui/view/__init__.py", line 173, in _input_loop
    self.on_exception(exc)
  File "/home/me/.pyenv/versions/3.11.3/lib/python3.11/site-packages/austin_tui/view/austin.py", line 67, in on_exception
    self.callback(self.Event.EXCEPTION, exc)
  File "/home/me/.pyenv/versions/3.11.3/lib/python3.11/site-packages/austin_tui/__main__.py", line 115, in on_view_event
    {
  File "/home/me/.pyenv/versions/3.11.3/lib/python3.11/site-packages/austin_tui/__main__.py", line 164, in on_exception
    raise exc
  File "/home/me/.pyenv/versions/3.11.3/lib/python3.11/site-packages/austin_tui/view/__init__.py", line 151, in _input_loop
    done, pending = await asyncio.wait(
                    ^^^^^^^^^^^^^^^^^^^
  File "/home/me/.pyenv/versions/3.11.3/lib/python3.11/asyncio/tasks.py", line 415, in wait
    raise TypeError("Passing coroutines is forbidden, use tasks explicitly.")
TypeError: Passing coroutines is forbidden, use tasks explicitly.
```

### Steps to Reproduce

1. Run `austin-tui test.py`
2. Hit any key such as an arrow key

**Expected behavior:** 

Keyboard navigation to work

**Actual behavior:**

Crash

**Reproduces how often:** 

100%

### Versions

You can get this information from copy and pasting the output of `austin
--version` from the command line. Also, please include the OS and what version
of the OS you're running.

Austin 3.5.0
Python 3.11.3
Linux 5.10.0-21-amd64 #1 SMP Debian 5.10.162-1 (2023-01-21) x86_64 GNU/Linux

### Additional Information

Any additional information, configuration or data that might be necessary to
reproduce the issue.

**Comment by kesmit13** [comment]
It appears that a naive solution is to change line 152 in `austin_tui/view/__init__.py` to the following fixes it (at least for Python 3.11).
```
                            [asyncio.ensure_future(_()) for _ in self._event_handlers[event]]
```

**Comment by P403n1x87** [comment]
@kesmit13 Thanks for the report. I believe this is due to Python 3.11 dropping support for coroutines in calls to `asyncio.wait`. Feel free to open a PR with the suggested fix 🙏

**Comment by kesmit13** [comment]
Fixed in #29
